killbill-integration-tests/core/usage/test_capacity.rb

```diff
@@ -168,10 +177,11 @@ def test_multiple_units
 
       check_invoice_no_balance(second_invoice, 5.00, 'USD', '2015-03-01')
       check_usage_invoice_item(second_invoice.items[0], second_invoice.invoice_id, 5.00, 'USD', 'USAGE', 'basic-monthly', 'basic-monthly-evergreen', 'basic-monthly-usage1', '2015-02-01', '2015-03-01')
-
+      check_invoice_item_detail(second_invoice.items[0],
+                                [{:tier => 2, :unit_type => 'bandwith-meg-sec', :existing_usage => 0, :unit_qty => 101, :tier_price => 5.0 },
+                                 {:tier => 2, :unit_type => 'members', :existing_usage => 0, :unit_qty => 10, :tier_price => 5.0 }], 5.0)
```

Comment:
@sbrossie I think we would want to see instead:

```
[{:tier => 2, :unit_type => 'bandwith-meg-sec', :existing_usage => 0, :unit_qty => 101, :tier_price => 5.0 },
 {:tier => 1, :unit_type => 'members', :existing_usage => 0, :unit_qty => 10, :tier_price => 1.0 }]
```

Thoughts?

Reply:
@pierre There are 2 issues to discuss here:
1. Whether the limit is inclusive or not -- basically whether `10` is part of `tier1` or `10` is part of `tier2` ?
2. Assuming `10` is part of `tier1` -- matching the assertion we see right above -- then i agree.

In any case there is something to fix here because the line above put this in  `tier1`